src/rpc/rpc_client.cpp

```
@@ -69,18 +77,22 @@ class RpcClientImpl : public transport::ITransportListener {
 
         running_ = false;
 
-        // Cancel all pending calls
+        std::vector<std::pair<RpcCallback, RpcResponse>> shutdown_cbs;
         {
```

Cppcheck (2.20.0)

[style] The function 'get_protocol_version' is never used.

(unusedFunction)